src/openrct2-ui/windows/Guest.cpp

```diff
@@ -2067,6 +2067,12 @@ void window_guest_debug_paint(rct_window* w, rct_drawpixelinfo* dpi)
     auto peep = GET_PEEP(w->number);
     auto x = w->x + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].left + 4;
     auto y = w->y + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].top + 4;
+    {
+        //args only holds one thing, so no array
+        int32_t args = {peep->sprite_index};
+        gfx_draw_string_left(dpi, STR_PEEP_DEBUG_SPRITE_INDEX, args, 0 x, y);
```

Comment:
`gfx_draw_string_left` takes a pointer to an argument list, not a single `int32_t`. You can use the `common_format_args` if you do not wish to make a new array here.

Reply:
im not sure i understand. what is ```common_format_args```? I can't find any reference to it in the code

Reply:
`set_format_arg(0, uint16_t, peep->sprite_index);`

Then replace the args in function call with `gCommonFormatArgs`.